docs/news.rst

```diff
@@ -2,6 +2,39 @@
 Release Notes
 =============
 
+2.3.1 (2021-04-XX)
+==================
+
+Highlights
+----------
+
+- Entities can be annotated with boolean datatypes, e.g.:
```

Comment:
I'd suggest replacing "e.g." with "for example" just to avoid the double punctuation of having a period followed by a colon. I am also removing latin abbreviations from docs.synapse.org as I find them, since they are not as easily understood by all audiences as the plain English equivalent.